packages/manager/apps/pci-volume-snapshot/src/core/HidePreloader.tsx

```diff
@@ -0,0 +1,12 @@
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
```

Comment:
Maybe we can imagine rework these two components (HidePreloader & ShellRoutingSync) that are not real components and turn them into hooks.

We have made this transformation into future `pci-instances` µapp, I can push them into `pci-common` if you want :)